tests/images/polar_alignment_utils_new.py

```diff
@@ -0,0 +1,60 @@
+from PIL import Image, ImageOps
+
+
+def fix_image_size(max_width, max_height, image):
+    """Resizes images while maintaining the same aspect ratio.
+
+    Args:
+        max_width: Image width
+        max_height: Image height
+        image: Image to be resized
+
+    Returns:
+        The resized image.
+    """
+    width_ratio = max_width / image.size[0]  # resized image ratios
+    height_ratio = max_height / image.size[1]
+
+    new_width = int(width_ratio * image.size[0])  # proportional width, height value calculations
+    new_height = int(height_ratio * image.size[1])
+
+    new_image = image.resize((new_width, new_height))
+    return new_image
+
+
+def grid_display(
+    background_jpg,
+    overlay_png,
+    width=4800,
+    height=2400,
+    alpha=0.2,
+    final_im_filename="final_output.png",
```

Comment:
Keep this slightly generic, as per above. 

I would also lean toward giving it the same filename as the incoming but with some added info.  You would do that by settng the default to `None` here and then at top of function do something like:

```
if output_filename is None:
    output_filename = background_filename.replace('.jpg', '-overlay.png')
```

I had mentioned this briefly in the code review the other day I think.